deploy/operator/utils.sh

```diff
@@ -47,6 +47,20 @@ function wait_for_pods(){
   echo "Pods in namespace $1 are ready"
 }
 
+function wait_for_deployment() {
+    deployment="$1"
+    namespace="${2:-}"
+    condition="Available"
+
+    echo "Waiting for (deployment) on namespace (${namespace}) with name (${deployment}) to be created..."
+    for i in {1..40}; do
+        oc get deployment "${deployment}" --namespace=${namespace} |& grep -ivE "(no resources found|not found)" && break || sleep 10
+    done
+
+    echo "Waiting for (deployment) on namespace (${namespace}) with name (${deployment}) to become (${condition})..."
+    oc wait -n "${namespace}" --for=condition=${condition} ${object} ${deployment} --timeout=${timeout}
```

Comment:
That would ensure that one replica rolled out. Lets use 
```
oc rollout status deploy/${deployment} -n "${namespace}" --timeout=${timeout}
```
here too. Its probably worth replacing `oc wait` entirely